pkg/config/engine/containerd/config_v1.go

```diff
@@ -165,3 +158,9 @@ func (c *ConfigV1) GetRuntimeConfig(name string) (engine.RuntimeConfig, error) {
 		tree: runtimeData,
 	}, nil
 }
+
+func (c *ConfigV1) EnableCDI() {
+	config := *c.Tree
+	config.SetPath([]string{"plugins", "cri", "containerd", "enable_cdi"}, true)
```

Comment:
here too c.Logger.Infof("enabled CDI in cri and containerd")

Reply:
If we want to add a debug or log statement, it should be at the call site.